pom.xml

```diff
@@ -278,7 +278,7 @@
              and to our own explicit dependencies in the relevant artifacts.
           -->
         <version.io.smallrye.jandex>3.0.5</version.io.smallrye.jandex>
-        <version.org.hibernate.commons.annotations.orm6>6.0.5.Final</version.org.hibernate.commons.annotations.orm6>
+        <version.org.hibernate.commons.annotations.orm6>6.0.6.Final</version.org.hibernate.commons.annotations.orm6>
```

Comment:
Not okay to do this. We need to be aligned on ORM.

Reply:
+1 that is actually realigning it back (https://github.com/hibernate/hibernate-orm/blob/e8f96761378720e7fcc2e84113ccca720c89f2a9/settings.gradle#L61) 
we have the correct version in our main branch and in `wip/main/dependency-update/orm6-in-main-code`  we have several commits upgrading orm and annotation versions, so it ends up being incorrect 🙈